docs/supported.md

```diff
@@ -194,57 +220,63 @@ $\displaystyle\sum_{\mathclap{1\le i\le j\le n}} x_{ij}$ `\sum_{\mathclap{1\le i
 
-`\kern`, `\mkern`, and `\hspace` accept unbraced distances, as in: `\kern1em`.
+`\kern`, `\mkern`, `\mskip`, and `\hspace` accept unbraced distances, as in: `\kern1em`.
 
 `\mkern` and `\mskip` will not work in text mode and both will write a console warning for any unit except `mu`.
 
+<div class="katex-hopscotch">
+
 ## Logic and Set Theory
 
 |||||
-|:--------------------|:--------------------------|:----------------------------------|:-----
-|$\forall$ `\forall`  |$\complement$ `\complement`|$\therefore$ `\therefore` |$\neg$ `\neg` or `\lnot`
-|$\exists$ `\exists`  |$\subset$ `\subset`  |$\because$ `\because`  |$\emptyset$ `\emptyset`
-|$\nexists$ `\nexists`|$\supset$ `\supset`  |$\mapsto$ `\mapsto` |$\varnothing$ `\varnothing`
-|$\in$ `\in` |$\mid$ `\mid`  |$\to$ `\to`|$\implies$ `\implies`
-|$\notin$ `\notin` |$\land$ `\land`|$\gets$ `\gets`  |$\impliedby$ `\impliedby`
-|$\ni$ `\ni` |$\lor$ `\lor`  |$\leftrightarrow$ `\leftrightarrow`|$\iff$ `\iff`
-|$\notni$ `\notni` |
+|:--------------------|:--------------------------|:----------------------------|:-----
+|$\forall$ `\forall`  |$\complement$ `\complement`|$\therefore$ `\therefore`    |$\emptyset$ `\emptyset`
+|$\exists$ `\exists`  |$\subset$ `\subset`  |$\because$ `\because`              |$\empty$ `\empty`
+|$\exist$ `\exist`    |$\supset$ `\supset`  |$\mapsto$ `\mapsto`                |$\varnothing$ `\varnothing`
+|$\nexists$ `\nexists`|$\mid$ `\mid`        |$\to$ `\to`                        |$\implies$ `\implies`
+|$\in$ `\in`          |$\land$ `\land`      |$\gets$ `\gets`                    |$\impliedby$ `\impliedby`
+|$\isin$ `\isin`      |$\lor$ `\lor`        |$\leftrightarrow$ `\leftrightarrow`|$\iff$ `\iff`
+|$\notin$ `\notin`    |$\ni$ `\ni`          |$\notni$ `\notni`                  |$\neg$ `\neg` or `\lnot`
 
-Direct Input:`∀ ∴ ∁ ∵ ∃ ∣ ∈ ∉ ∋ ⊂ ⊃ ∧ ∨ ↦ → ← ↔ ¬ ℂ ℍ ℕ ℙ ℚ ℝ ℤ`
+Direct Input: $∀ ∴ ∁ ∵ ∃ ∣ ∈ ∉ ∋ ⊂ ⊃ ∧ ∨ ↦ → ← ↔ ¬$ ℂ ℍ ℕ ℙ ℚ ℝ
 
 ## Macros
 
 |||
 |:-------------------------------------|:------
 |$\def\foo{x^2} \foo + \foo$           | `\def\foo{x^2} \foo + \foo`
 |$\gdef\bar#1{#1^2} \bar{y} + \bar{y}$ | `\gdef\bar#1{#1^2} \bar{y} + \bar{y}`
-|$~$                                   | `\global\def\macroname#1#2…{definition}`
-|$~$                                   | `\newcommand\macroname[numargs]{definition}`
-|$~$                                   | `\renewcommand\macroname[numargs]{definition}`
-|$~$                                   | `\providecommand\macroname[numargs]{definition}`
+|                                      | `\global\def\macroname#1#2…{definition}`
+|                                      | `\newcommand\macroname[numargs]{definition}`
+|                                      | `\renewcommand\macroname[numargs]{definition}`
+|                                      | `\providecommand\macroname[numargs]{definition}`
 
 Macros can also be defined in the KaTeX [rendering options](options.md).
 
```

Comment:
The next line says "up to ten arguments".  That should really be nine, as we start counting at 1.